xbmc/Application.cpp

```diff
@@ -3085,6 +3085,10 @@ void CApplication::OnPlayerCloseFile(const CFileItem &file, const CBookmark &boo
   bool playCountUpdate = false;
   float percent = 0.0f;
 
+    // Make sure we don't reset existing bookmark etc. on eg. player start failure
+  if (bookmark.timeInSeconds == 0.0f)
```

Comment:
Mind fixing the indentation of the comment?

Reply:
@ksooo : Of course not ;-) Done...

Reply:
Thx